services/src/main/java/org/fao/geonet/api/selections/SelectionsApi.java

```diff
@@ -179,4 +196,71 @@ ResponseEntity<Integer> clear(
 
         return new ResponseEntity<>(nbSelected, HttpStatus.OK);
     }
+
+    @io.swagger.v3.oas.annotations.Operation(summary = "Get current selection resource types")
+    @RequestMapping(
+        method = RequestMethod.GET,
+        value = "/{bucket}/statistics",
```

Comment:
Maybe, use a search with aggregation on `resourceType` directly?
Also I'm not 100% sure it is needed, because `resourceType` seems not used in https://github.com/geonetwork/core-geonetwork/pull/8706/files#diff-54787565d74f138cb8569205e93acd3fd0df6c970043649eb28844f4d0b0995aR444-R449 ?